loader/vk_loader_platform.h

```diff
@@ -170,6 +175,26 @@ static inline char *loader_platform_executable_path(char *buffer, size_t size) {
     buffer[ret] = '\0';
     return buffer;
 }
+#elif defined(__DragonFly__) || defined(__FreeBSD__) || defined(__NetBSD__)
```

Comment:
I see two platforms here that aren't mentioned anywhere else. `__DragonFly__` and `__NetBSD__`. Certainly its fine to support more platforms but it makes me curious if dragonfly & netbsd only need this specific version of this function or if their platforms needs more than this. Not a blocking issue.

Reply:
DragonFly currently uses FreeBSD **downstream** patches as is due to largerly the same userland API and the same packaging system (build recipes but not binaries). Compared to this PR those patches reduce proliferation of excessively long platform allow-lists using POSIX as a common ground.

https://github.com/freebsd/freebsd-ports/tree/main/graphics/vulkan-loader/files
https://github.com/DragonFlyBSD/DPorts/tree/master/graphics/vulkan-loader/files
https://github.com/DragonFlyBSD/DeltaPorts/tree/master/ports/graphics/vulkan-loader/files
https://sting.dragonflybsd.org/dports/logs/graphics___vulkan-loader.log

OpenBSD probably used FreeBSD patches with some additions. NetBSD is WIP probably due to not yet ported Vulkan drivers or kernel DRM prerequisites.

https://github.com/openbsd/ports/tree/master/graphics/vulkan-loader/patches
https://github.com/NetBSD/pkgsrc-wip/tree/master/vulkan-loader/patches

Reply:
Sounds like a "Its there because it makes your life easier". Thanks for the expanded info!